deploy/settings.py

```diff
@@ -0,0 +1,205 @@
+"""
+Django settings for tr_sys project.
+
+Generated by 'django-admin startproject' using Django 1.11.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.11/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.11/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.11/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'a=()q#x6$s_2tnzgvoudd=5&gbv*57bcr(^%f)&rp-7ydf(9ao'
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = ['ALLOWED_HOSTS_VALUE']
+
+
+# Application definition
+
+INSTALLED_APPS = [
```

Comment:
Can you import settings.py from https://github.com/mapleknight/Relay/blob/master/tr_sys/tr_sys/settings.py and only update the variables that you need?

Reply:
@southalln The settings.py in deploy folder is a template file.If we can use this template file to replace the tr_sys/tr_sys/settings.py, we can keep only one copy of it.